src/apim.runtime.module.ts

```diff
@@ -92,9 +92,10 @@ import { TraceClick } from "./bindingHandlers/traceClick";
 import { GraphqlService } from "./services/graphqlService";
 import { ListOfApisRuntimeModule } from "./components/apis/list-of-apis/listOfApis.runtime.module";
 import { ProductListRuntimeModule } from "./components/products/product-list/productList.runtime.module";
-import { ProductApisRuntimeModule } from "./components/products/product-apis/productApis.runtime.module";
 import { OperationListRuntimeModule } from "./components/operations/operation-list/operationList.runtime.module";
 import { DetailsOfApiRuntimeModule } from "./components/apis/details-of-api/detailsOfApi.runtime.module";
+import { OperationDetailsRuntimeModule } from "./components/operations/operation-details/operationDetails.runtime.module";
+import { ProductApisRuntimeModule } from "./components/products/product-apis/productApis.runtime.module";
```

Comment:
I don't see you binding this module, am I missing something?

Reply:
It is just moved from line 95.

Reply:
ooh got it